pyfar/dsp/filter/fractional_octaves.py

```diff
@@ -240,6 +248,41 @@ def fractional_octave_bands(
         >>> ax.set_title(
         ...     "Filter bands and the sum of their squared magnitudes")
 
+    Show that the filters meet class I tolerance according to IEC 61260-1
+
+    .. plot::
+
+        >>> import pyfar as pf
+        >>> import numpy as np
+        >>> import matplotlib.pyplot as plt
+        >>>
+        >>> # get octave filter
+        >>> center_frequency = 1000
+        >>> octave_filter = pf.dsp.filter.fractional_octave_bands(
+        >>>     pf.signals.impulse(2**12), 1, None,
+        ...     (center_frequency, center_frequency))
```

Comment:
This is a good use-case for the impulse response method I would say. 
It should get rid of the arbitrarily chosen length.